pages/services/cassandra/2.6.0-3.11.4/configuration/index.md

```diff
@@ -0,0 +1,281 @@
+
+### Launch two {{ model.techShortName }} clusters
+
+1. Launch the first cluster with the default configuration:
+
+```shell
+dcos package install {{ model.packageName }}
+```
+
+2. Create an `options.json` file for the second cluster that specifies a different service name and data center name:
+
+```json
+{
+  "service": {
+    "name": "{{ model.serviceName }}2",
+    "data_center": "dc2"
+  }
+}
+```
+
+3. Launch the second cluster with these custom options:
+```
+dcos package install {{ model.packageName }} --options=<options.json>
+```
+
+### Get the seed node IP addresses
+
+<p class="message--note"><strong>NOTE: </strong>If your {{ model.techShortName }} clusters are not on the same network, you must set up a proxying layer to route traffic.</p>
+
+1. Get the list of seed node addresses for the first cluster:
+
+```shell
+dcos {{ model.packageName }} --name={{ model.serviceName }} endpoints node
+```
+
+Alternatively, you can get this information from the scheduler HTTP API:
+
+```json
+DCOS_AUTH_TOKEN=$(dcos config show core.dcos_acs_token)
+DCOS_URL=$(dcos config show core.dcos_url)
+curl -H "authorization:token=$DCOS_AUTH_TOKEN" $DCOS_URL/service/{{ model.serviceName }}/v1/endpoints/node
+```
+
+Your output will resemble:
+
+```
+{
+  "address": [
+    "10.0.1.236:9042",
+    "10.0.0.119:9042"
+  ],
+  "dns": [
+    "node-0-server.{{ model.serviceName }}.autoip.dcos.thisdcos.directory:9042",
+    "node-1-server.{{ model.serviceName }}.autoip.dcos.thisdcos.directory:9042"
+  ],
+  "vip": "node.{{ model.serviceName }}.l4lb.thisdcos.directory:9042"
+}
+```
+
+Note the IPs in the `address` field.
+
+2. Run the same command for your second {{ model.techShortName }} cluster and note the IPs in the `address` field:
+
+```
+dcos {{ model.packageName }} --name={{ model.serviceName }}2 endpoints node
+```
+
+### Update configuration for both clusters
+
+1. Create an `options2.json` file with the IP addresses of the first cluster (`{{ model.serviceName }}`):
+
+```json
+{
+  "service": {
+    "remote_seeds": "10.0.1.236:9042,10.0.0.119:9042"
+  }
+}
+```
+
+2. Update the configuration of the second cluster:
+
+```
+dcos {{ model.packageName }} --name={{ model.serviceName}}2 update start --options=options2.json
+```
+
+Perform the same operation on the first cluster, creating an `options.json` which contains the IP addresses of the second cluster (`{{ model.serviceName }}2`)'s seed nodes in the `service.remote_seeds` field. Then, update the first cluster's configuration: `dcos {{ model.packageName }} --name={{ model.serviceName }} update start --options=options.json`.
+
+Both schedulers will restart after each receives the configuration update, and each cluster will communicate with the seed nodes from the other cluster to establish a multi-data-center topology. Repeat this process for each new cluster you add.
```

Comment:
```suggestion
Both schedulers will restart after each receives the configuration update, and each cluster will communicate with the seed nodes from the other cluster to establish a multi-data center topology. Repeat this process for each new cluster you add.
```

Reply:
@viivek46 Please be specific on the list of steps that an user must repeat for each new cluster to be added.